usmqe_tests/api/gluster/test_gluster_cluster.py

```diff
@@ -71,19 +58,35 @@ def test_cluster_create_valid(
     nodes = []
     provisioner_ip = None
     network = None
+    node_ids = []
     for x in valid_nodes:
         if "tendrl/server" in x["tags"]:
             network = "{}/22".format(
-                ast.literal_eval(x["networks"][net_interface]["ipv4"])[0])
+                ast.literal_eval(x["networks"][network_interface]["ipv4"])[0])
             continue
-        ips = ast.literal_eval(x["networks"][net_interface]["ipv4"])
+        ips = ast.literal_eval(x["networks"][network_interface]["ipv4"])
```

Comment:
I understand this is that infamous ugly workaround for [RHBZ 1455816](https://bugzilla.redhat.com/show_bug.cgi?id=1455816), but what would happen when the bug is fixed? Would we notice that immediately, without wasting time on debugging weird error?